Chap_API_Server.tex

```diff
@@ -1300,7 +1300,21 @@ \subsection{\code{pmix_server_fencenb_fn_t}}
 \adviceimplend
 
 \advicermstart
-The host will receive a single call for each collective operation. It is the responsibility of the host to identify the nodes containing participating processes, execute the collective across all participating nodes, and notify the local \ac{PMIx} server library upon completion of the global collective.
+The host will receive a single call for each collective operation. It is the responsibility of the host to identify the nodes containing participating processes, execute the collective across all participating nodes, and notify the local \ac{PMIx} server library upon completion of the global collective. Data received from each node \emph{shall} be simply concatenated to form an aggregated unit, as shown in the following example:
```

Comment:
It seems that we do not emphasis *must* in the rest of the document so we should not do it here.